src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -829,6 +829,78 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(topBottomContainerViewController)
 	}
 
+	// MARK: TestCertificateInfo
+
+	private func showTestCertificateInfo() {
+		let testCertificateInfoViewController = ExposureSubmissionTestCertificateInfoViewController(
+			ExposureSubmissionTestCertificateViewModel(
+				testType: .pcr,
+				presentDisclaimer: { [weak self] in
+					let detailViewController = HTMLViewController(model: AppInformationModel.privacyModel)
+					detailViewController.title = AppStrings.AppInformation.privacyTitle
+					detailViewController.isDismissable = false
+					if #available(iOS 13.0, *) {
+						detailViewController.isModalInPresentation = true
+					}
+					self?.push(detailViewController)
+				}
+			),
+			showCancelAlert: { [weak self] in
+				self?.showEndRegistrationAlert(
+					submitAction: UIAlertAction(
+						title: AppStrings.ExposureSubmission.TestCertificate.Info.Alert.ok,
+						style: .default,
+						handler: { _ in
+							self?.navigationController?.dismiss(animated: true)
+						}
+					)
+				)
+			},
+			didTapPrimaryButton: { testType, optionalBirthDateString in
+				Log.debug("NYD - \(testType), \(String(describing: optionalBirthDateString))")
+			},
+			didTapSecondaryButton: { [weak self] in
+				self?.navigationController?.dismiss(animated: true)
```

Comment:
This will register the test then, right? Could we already implement this here, to register it with the loading indicator and error message handling?

Reply:
yes, this the hook to continue the flow. Let's have a talk about what are the next steps here so we can wire it up.